docking_bundle_name:  DOCKING_TAGS
left_bundle_tag:  LEFT-TAG
right_bundle_tag: RIGHT-TAG
alignment_tag: TAG
odom_frame: odom
base_link_frame: base_link


#Docking Tuning Parameters 
stage_1_dist: 0.8 #Stage 1 Distance from Docking Station    
stage_2_dist: 0.6 #Stage 2 Distance from Docking Station
stage_3_dist: 0.5 #Stage 3 Distance from Docking Station
engage_dist: 0.095 #Final engagement distance

stage_1_radius_tolerance: 0.03 #Radial tolerance of stage 1 position from Docking Station 
stage_2_radius_tolerance: 0.03 #Radial tolerance of stage 2 position from Docking Station
stage_3_radius_tolerance: 0.025 #Radial tolerance of stage 3 position from Docking Station

stage_alignment_tolerance: 0.01 #Alignment tolerance to staging position (Rads)
final_alignment_tolerance: 0.008 #Final alignment tolerance (Rads)

pid_staging_alignment_gain: 0.6
pid_staging_alignment_output_limit: 0.2

pid_staging_tag_detect_gain: 0.2
pid_staging_tag_detect_output_limits: 0.2

pid_staging_angular_gain: 0.3
pid_staging_angular_output_limits: 0.1

pid_staging_linear_gain: 0.5
pid_staging_linear_output_limits: 0.08

pid_final_alignment_gain: 0.5
pid_final_alignment_output_limit: 0.05

#Script Efficiencies
tag_detect_ratio: 2 #Ratio at which node checks for tag detections as a function of published frames. 
odom_update_ratio: 0.5 #Ratio at which node checks for tag detections as a function of published frames. 
